vscode/src/workspace.ts

```diff
@@ -44,7 +44,11 @@ export class Workspace implements WorkspaceInterface {
     this.isMainWorkspace = isMainWorkspace;
 
     this.registerRestarts(context);
-    this.registerRebaseWatcher(context);
+    this.registerCreateDeleteWatcher(
+      context,
+      ".git/{rebase-merge,rebase-apply}",
```

Comment:
I tried find a list of all the various temporary files that git creates, for example during cherry-picking there's `CHERRY_PICK_HEAD`, but couldn't find a comprehensive list.